src/components/App.jsx

```diff
@@ -238,7 +243,7 @@ class App extends Component {
 		let group;
 
 		let contents = (
-			<Button color="inherit" href={"login"} className={classes.smallButton}>
+			<Button color="inherit" onClick={() => browserHistory.push("/Login")} className={classes.smallButton}>
```

Comment:
Is it a typo or we change this route to be capitalized "Login"?

Reply:
I do not recall whether it was this PR or from earlier...

Reply:
It was there from before for all the components